README.md

```diff
@@ -116,6 +112,14 @@ Additional resources:
 * [serverfull](https://github.com/asecurityteam/serverfull)
 * [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
 
+### Environment Variables
+Here are the environment variables that need to be set
+
+| Name                  | Required | Description                                                                          | Example                       |
+|-----------------------|:--------:|--------------------------------------------------------------------------------------|-------------------------------|
+| NEXPOSE_API_USERNAME  |   Yes    | Username to access the Nexpose instance                                              | myusername                    |
```

Comment:
task: We should definitely call out that folks will need to handle the basic auth injection for requests. We should also suggest to them that they use our transportd project for this. However, we shouldn't prescribe the names of the variables they use for this case because we can't guarantee they will both use transport and use our sample OpenAPI specification as-is.